src/Backend/Core/Js/ckfinder/core/connector/php/CKSource/CKFinder/Acl/Acl.php

```diff
@@ -19,28 +19,28 @@
 
 /**
  * The Acl class.
- * 
```

Comment:
you should revert the changes to everything within the ckfinder folder. It is an external library so we'd prefer to keep it the way it was released